coral/src/app/router_utils.ts

```diff
@@ -1,5 +1,13 @@
 import isString from "lodash/isString";
 
+enum Routes {
```

Comment:
I really like this approach of having a central map for such constants ^^

Reply:
Me too! We can have more maps / utils for routing in the end. I started to get down a rabbit hole a bit last week but stopped because it's not really worth it at the moment - the main nav has special requirements right now with linking in the react app and outside of it etc.